src/locales/en.json

```diff
@@ -76,7 +76,7 @@
     "primary_faq": "Manuel d'utilisation",
     "primary_lock_screen": "Unlock screen",
     "primary_logout": "Déconnexion",
```

Comment:
there is a lot of french word in this feature. Is it intended?

Reply:
Not really, I don't have the translations at this point unless I missed it

Reply:
ping @poupotte

Reply:
@Crash-- we talked about it in a meeting, we'll review these keys/translations in another PR